src/test/java/io/anserini/analysis/DefaultEnglishAnalyzerTest.java

```diff
@@ -0,0 +1,29 @@
+package io.anserini.analysis;
+
+import org.apache.lucene.analysis.CharArraySet;
+import org.apache.lucene.analysis.en.EnglishAnalyzer;
+import org.junit.Test;
+
+import java.util.Arrays;
+
+import static io.anserini.analysis.DefaultEnglishAnalyzer.fromArguments;
+import static org.junit.Assert.assertEquals;
+
+public class DefaultEnglishAnalyzerTest {
+
+    @Test
```

Comment:
two space indent to be consistent with rest of code base?

Reply:
Oops, fixed! I had also forgotten the license header.